### PR TITLE
Reduce errors & not require all targets

### PR DESCRIPTION
**Pull request by arnasfomenko**
## Returns console warning if domTarget isn't found
Screenshot 2022-11-16 at 11 09 11

## Other viewers in the page doesn't break anymore if one of the viewers is not available
Screenshot 2022-11-16 at 11 10 12

## Reduces some other random console errors, when targets are not found, not available

### CODE REVIEW

File: src/components/arwrapper.js [hunk 1]
```diff
@@ -17,27 +17,31 @@ function placeholderSrc() {
 }
 
 export default function ARWrapper(viewerUrl, gltfUrl, usdzUrl, target) {
+  if (!target) {
```
Comment (mvaldas9) [inline src/components/arwrapper.js:20]: if we are using early return pattern here, do we need other checks below?

File: src/components/metadata.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ function Metadata(image, gltf, usdz, name, target) {
   script.type = "application/ld+json"
   script.text = data.replace(/^\s*[\r\n]/gm, '') // Remove empty lines caused by missing params
 
-  target.appendChild(script)
+  target?.appendChild(script)
```
Comment (mvaldas9) [inline src/components/metadata.js]: I guess we could return early in this file as well? Before creating script element in memory, etc.

**Comment by mvaldas9** [comment]
- In some places early return pattern could be used instead of "random" `?`, looks nicer imho
  - In same places I was not sure how `tempLink?` could be undefined, but having said that I didn't test this code, so you might know better
- I guess let's increase package version number?

**Review by BenasLengvinasCgTrader — APPROVED** [review]

**Comment by arnasfomenko** [comment]
@mvaldas9 
Refactored some parts, added early returns. Will bump version before release

**Review by mvaldas9 — APPROVED** [review]

**Pull request merged**